### DISCUSSION TITLE
Unable to access `abuja@djangogirls.org` address

### DISCUSSION THREAD

**Discussion by TobeTek** [DjangoGirls/djangogirls#919] [Q&A]
Hello Django Girls Team,

I do hope this email finds you well.
We're quite excited to be organizing Django Girls Abuja! Unfortunately, we've hit a few challenges accessing our Django Girls account.

1. We can't access the [abuja@djangogirls.org](mailto:abuja@djangogirls.org) inbox. We can't find login details in any of the acceptance emails sent to the organizer email addresses. We reached out to the previous organizers to get the password they used, but when signing in on Gmail, we get the error *"Couldn't find your Google account".*
Could you kindly provide login credentials to the [abuja@djangogirls.org](mailto:abuja@djangogirls.org) email?

2. A bit auxiliary, but we have been unable to join the DjangoGirls Slack community. Could you provide an invite if possible?

Thank you very much, and I look forward to hearing from you.

Best regards,
Emmanuel Katchy

P.S: Tagging for visiblity, @amakarudze, @das-g  🙏🏾

**Comment by vanessa** [comment]
@TobeTek is this still an issue?

**Comment by TobeTek** [comment]
Hi @vanessa  👋
I don't think it is. The Google account exists now
